src/main/java/pl/isa/biblioteka/BorrowBooks.java

```diff
@@ -1,14 +1,13 @@
 package pl.isa.biblioteka;
 
-import java.util.List;
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Osobiście nie jestem fanem * w importach, one i tak są automatycznie dodawane i normalnie prawie nikt ich nie czyta poza narzędziem do statycznej analizy kodu ;)

Reply:
InteliiJ sam mi to zamienił ale już pobrałem tylko poprawne importy